src/js/components/dataDives/sections/MainCards.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * MainCards.jsx
+ * Created by Nick Torres 06/17/22
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { largeScreen } from 'dataMapping/shared/mobileBreakpoints';
+import ExternalLink from 'components/sharedComponents/ExternalLink';
+import { FlexGridRow, FlexGridCol } from 'data-transparency-ui';
+import ReadMoreUpdated from '../../sharedComponents/ReadMoreUpdated';
+import EquityMainCard from '../../sharedComponents/EquityMainCard';
+
+const propTypes = {
+    contentObject: PropTypes.object.isRequired
+};
+
+const MainCards = ({ contentObject }) => {
+    const additionalFunctionality = (expanded) => {
+        if (window.innerWidth >= largeScreen) {
+            const toExpand = document.querySelector(".card__toExpand");
+            if (!expanded) {
+                toExpand.style.paddingBottom = "116px";
+            }
+            else {
+                toExpand.style.paddingBottom = "24px";
+            }
+        }
+    };
+
+    const bowieImg = <img className="main-cards__svg" src="../../../../img/top-bowie-state-combined-image.svg" alt="" />;
```

Comment:
these should have the role="presentation" attribute since these are presentation only

Reply:
addd that for all the images